tradeit-android-sdk/src/main/java/it/trade/android/sdk/manager/TradeItLinkedBrokerManager.java

```diff
@@ -44,11 +45,23 @@
     protected TradeItBrokerLinker brokerLinker;
 
     private List<TradeItLinkedBroker> linkedBrokers = new ArrayList<>();
-    private Context context = TradeItSDK.getContext();
-    private String apiKey =  TradeItSDK.getApiKey();
-    private TradeItEnvironment environment = TradeItSDK.getEnvironment();
+    private final Context context;
+    private final String apiKey;
+    private final TradeItEnvironment environment;
+
+    public TradeItLinkedBrokerManager(Context context) throws TradeItKeystoreServiceCreateKeyException, TradeItRetrieveLinkedLoginException {
+        this.context = context;
+        this.apiKey =  TradeItSDK.getApiKey();
```

Comment:
This would just throw an exception as the use of the new class `TradeItSdk2` means `TradeItSDK.configure(...)` is never called. In addition, everywhere else that references the `TradeItSDK` in this PR and the rest of the library would now crash.